akka-persistence-tck/src/main/scala/akka/persistence/scalatest/MayVerb.scala

```diff
@@ -41,7 +41,7 @@ trait MayVerb {
      * @see <a href="https://www.rfc-editor.org/rfc/rfc2119.txt">RFC 2119</a>
      */
     def may(right: => Unit)(implicit fun: StringVerbBlockRegistration, pos: Position): Unit = {
-      fun(leftSideString, "may", pos, right _)
+      fun(leftSideString, "may", pos, () => right)
```

Comment:
fix by `fix.scala213.ExplicitNullaryEtaExpansion` (logic in `DottyMigrate` rule)